frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radigoram-card-content-missing-transfer-connection/radigoram-card-content-missing-transfer-connection.component.html

```diff
@@ -0,0 +1,6 @@
+<ng-container *ngIf="transferPointName$ | async as transferPointName">
+    <h5 class="card-title">Fehlende Verbindung</h5>
+    <span class="card-text"
+        >Es fehlt eine Verbindung zu diesem Transferpunkt: </span
+    ><span class="badge bg-secondary p-1">{{ transferPointName }}</span>
+</ng-container>
```

Comment:
My understanding of the purpose of the radiogram is to communicate missing knowledge about the location in the domain. This text, however, looks more like an error message indicating a misconfigured exercise. It is an avoidable burden for  the signaler to translate it to the domain.

Reply:
I've reworded it :)